src/02-crm/contacts.apib

```diff
@@ -227,6 +227,7 @@ Update a contact.
         + tags: prospect, expo (array[string], optional)
         + custom_fields (array[CustomFieldValue], optional)
         + marketing_mails_consent: false (boolean, optional)
+        + `national_identification_number`: BE 1290 (string)
```

Comment:
PP: it would be better to move this property somewhere after IBAN and BIC, before the more "preferency" or metadata properties.